api/commands/run.ts

```diff
@@ -76,21 +67,24 @@ export class Runner {
         this.cancel();
       }, this.graph.runConfig.timeoutMillis);
     }
-    return this.resultPromise();
```

Comment:
wat, this is super weird

Reply:
you mean removing this? or what?

my changes here constitute some API cleanup, driven really by tslint warnings.

Reply:
I mean returning `this` - I've never seen that done before.